demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2162,17 +2162,22 @@ export class DemoMeetingApp
     // Also note that Firefox has its own device picker, which may be useful
     // for the first device selection. Subsequent device selections could use
     // a custom UX with a specific device id.
-    this.audioVideo.setDeviceLabelTrigger(
-      async (): Promise<MediaStream> => {
-        if (this.isRecorder() || this.isBroadcaster()) {
-          throw new Error('Recorder or Broadcaster does not need device labels');
+    if(!this.defaultBrowserBehaviour.doesNotSupportMediaDeviceLabels())
+    {
+      this.audioVideo.setDeviceLabelTrigger(
```

Comment:
Hmm I am not sure I understand the logic here. Basically you do not call setDeviceLabelTrigger for AndroidWebview. However, I think in that case, we just use the default label trigger which is the same here (See https://github.com/aws/amazon-chime-sdk-js/blob/master/src/devicecontroller/DefaultDeviceController.ts#L139). Is that correct?

Reply:
That's correct, the key fix here is that it no longer sends the user back to flow-need-permission and flow-devices when the device labels are found to be empty after calling enumerate devices.

Reply:
Great detail. Can we update the PR description about the problem details then?

Reply:
Added into the description!